### PR TITLE
Add initial molecule testing

### PR DESCRIPTION
**Pull request by JakobPetersson**

### CODE REVIEW

**Comment by smuda** [comment]
When I'm running `molecule create`, I get the following error message:

```
An exception occurred during task execution. To see the full traceback, use -vvv. The error was: ModuleNotFoundError: No module named 'molecule'
fatal: [localhost]: FAILED! => {"changed": false, "module_stderr": "Traceback (most recent call last):\n  File \"/Users/john/.ansible/tmp/ansible-tmp-1674073424.874485-11042-181845835636937/AnsiballZ_vagrant.py\", line 107, in <module>\n    _ansiballz_main()\n  File \"/Users/john/.ansible/tmp/ansible-tmp-1674073424.874485-11042-181845835636937/AnsiballZ_vagrant.py\", line 99, in _ansiballz_main\n    invoke_module(zipped_mod, temp_path, ANSIBALLZ_PARAMS)\n  File \"/Users/john/.ansible/tmp/ansible-tmp-1674073424.874485-11042-181845835636937/AnsiballZ_vagrant.py\", line 47, in invoke_module\n    runpy.run_module(mod_name='ansible.modules.vagrant', init_globals=dict(_module_fqn='ansible.modules.vagrant', _modlib_path=modlib_path),\n  File \"<frozen runpy>\", line 226, in run_module\n  File \"<frozen runpy>\", line 98, in _run_module_code\n  File \"<frozen runpy>\", line 88, in _run_code\n  File \"/var/folders/90/g1xy5_5d6vv1vg_zvf833n1m0000gn/T/ansible_vagrant_payload_jurxh2vy/ansible_vagrant_payload.zip/ansible/modules/vagrant.py\", line 37, in <module>\nModuleNotFoundError: No module named 'molecule'\n", "module_stdout": "", "msg": "MODULE FAILURE\nSee stdout/stderr for the exact error", "rc": 1}
```

After running `pip3 install molecule ansible-core` it worked much better. Perhaps `ansible-core` is a missing dependency in Readme?

**Comment by smuda** [comment]
What do you think about running `molecule test` in a [github action for each PR](https://github.com/marketplace/actions/ansible-molecule)?

File: molecule/default/verify.yml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+---
+# This is an example playbook to execute Ansible tests.
+
+- name: Verify
+  hosts: all
+  gather_facts: false
+  tasks:
+  - name: Example assertion
```
[the remark this answers is not in the document]
Comment (JakobPetersson) [reply molecule/default/verify.yml:8]: Molecule allows you to perform tests after it has performed converge. As you have seen, there are no explicit tests yet. I will clean this up now to make the PR better.

The current real test is that molecule converge can be performed without errors and without reported "changes" by ansible a a second time. This is automatically done when running molecule test.

**Comment by JakobPetersson** [comment]
> What do you think about running `molecule test` in a [github action for each PR](https://github.com/marketplace/actions/ansible-molecule)?

I think that is a really good idea that needs to be looked into. I'm new to molecule as well. But I hope we can get there.

**Comment by JakobPetersson** [comment]
> When I'm running `molecule create`, I get the following error message:
> 
> ```
> An exception occurred during task execution. To see the full traceback, use -vvv. The error was: ModuleNotFoundError: No module named 'molecule'
> fatal: [localhost]: FAILED! => {"changed": false, "module_stderr": "Traceback (most recent call last):\n  File \"/Users/john/.ansible/tmp/ansible-tmp-1674073424.874485-11042-181845835636937/AnsiballZ_vagrant.py\", line 107, in <module>\n    _ansiballz_main()\n  File \"/Users/john/.ansible/tmp/ansible-tmp-1674073424.874485-11042-181845835636937/AnsiballZ_vagrant.py\", line 99, in _ansiballz_main\n    invoke_module(zipped_mod, temp_path, ANSIBALLZ_PARAMS)\n  File \"/Users/john/.ansible/tmp/ansible-tmp-1674073424.874485-11042-181845835636937/AnsiballZ_vagrant.py\", line 47, in invoke_module\n    runpy.run_module(mod_name='ansible.modules.vagrant', init_globals=dict(_module_fqn='ansible.modules.vagrant', _modlib_path=modlib_path),\n  File \"<frozen runpy>\", line 226, in run_module\n  File \"<frozen runpy>\", line 98, in _run_module_code\n  File \"<frozen runpy>\", line 88, in _run_code\n  File \"/var/folders/90/g1xy5_5d6vv1vg_zvf833n1m0000gn/T/ansible_vagrant_payload_jurxh2vy/ansible_vagrant_payload.zip/ansible/modules/vagrant.py\", line 37, in <module>\nModuleNotFoundError: No module named 'molecule'\n", "module_stdout": "", "msg": "MODULE FAILURE\nSee stdout/stderr for the exact error", "rc": 1}
> ```
> 
> After running `pip3 install molecule ansible-core` it worked much better. Perhaps `ansible-core` is a missing dependency in Readme?

Yes, I will make additions to the README.

**Pull request merged**